src/main/scala/fs2/kafka/KafkaConsumer.scala

```diff
@@ -117,6 +117,19 @@ sealed abstract class KafkaConsumer[F[_], K, V] {
     */
   def partitionedStream: Stream[F, Stream[F, CommittableMessage[F, K, V]]]
 
+  /**
+    * Overrides the fetch offsets that the consumer will use when reading the
+    * next message. If this API is invoked for the same partition more than once,
+    * the latest offset will be used. Note that you may lose data if this API is
+    * arbitrarily used in the middle of consumption to reset the fetch offsets.
+    *
+    * The stream obtained from [[stream]] or [[partitionedStream]] will raise
+    * an [[IllegalArgumentException]] if the provided offset is negative, or an
```

Comment:
Scaladoc generation is failing because it can't find links for these exceptions. Easiest is to change to backticks (e.g. `IllegalArgumentException`) for both of these.